src/main/java/Bundle.properties

```diff
@@ -1308,7 +1308,7 @@ dataset.privateurl.noPermToCreate=To create a Private URL you must have the foll
 file.count={0} {0, choice, 0#Files|1#File|2#Files}
 file.count.selected={0} {0, choice, 0#Files Selected|1#File Selected|2#Files Selected}
 file.selectToAddBtn=Select Files to Add
-file.selectToAdd.tipLimit=File upload limit is {0} bytes per file. 
+file.selectToAdd.tipLimit=File upload limit is {0} per file. 
```

Comment:
Does this edit remove the size unit from the message? Is it moved somewhere? Because I didn't see it added anywhere in the PR changes.

Reply:
That's what I figured, but didn't want to assume. Thanks.